network/proto/v1/network.proto

```diff
@@ -0,0 +1,259 @@
+syntax = "proto3";
+
+package messari.network.v1;
+
+
+message Network {
+    // Name of the network being indexed
+    string id = 1;
+  
+    // The verison of the schema
+    string schema_version = 2;
+  
+    // The version of the subgraph
+    string subgraph_version = 3;
+  
+    // The version of the methodology
+    string methodology_version = 4;
+  
+    // #### Quantitative Data ####
+  
+    // Number of cumulative unique authors (see type Author for explanation)
+    int64 cumulative_unique_authors = 5;
+  
+    // The total number of blocks in the blockchain
+    int64 block_height = 6;
+  
+    // Total difficulty to reach the current block
+    int64 cumulative_difficulty= 7;
```

Comment:
minor but might be helpful to have a linter to catch this kind of thing: `cumulative_difficulty=`